compatible/pom.xml

```diff
@@ -30,7 +30,7 @@
 
     <dependencies>
         <dependency>
-            <groupId>${project.groupId}</groupId>
+            <groupId>org.apache.seata</groupId>
```

Comment:
如果他不依赖org.apache.seata:seata-all,会影响应用使用吗?假设我使用一个io.seata:seata-all 2.1.0-SNAPSHOT
If he doesn't depend on org.apache.seata:seata-all, will it affect the application usage? Let's say I use an io.seata:seata-all 2.1.0-SNAPSHOT

Reply:
In my judgment, if the compatible module doesn't depend on org.apache.seata:seata-all, then it's not complete and users will have problems using it because of the missing classes. If he's missing classes, what's the difference between that and just using io.seata:seata-all and typing a version specifically to be used as a usedby tag?

Reply:
io.seata:seata-all won't release module package.